src/SprykerEco/Yves/Easycredit/Form/DataProvider/EasycreditDataProvider.php

```diff
@@ -27,12 +27,18 @@ public function getData(AbstractTransfer $quoteTransfer): QuoteTransfer
     /**
      * @param \Generated\Shared\Transfer\QuoteTransfer $quoteTransfer
      *
-     * @return array
+     * @return array<array-key, string|null>
      */
     public function getOptions(AbstractTransfer $quoteTransfer): array
     {
+        $easycreditLegalText = null;
+        $easycreditLegalTextTransfer = $quoteTransfer->getEasycreditLegalText();
+        if ($easycreditLegalTextTransfer) {
+            $easycreditLegalText = $easycreditLegalTextTransfer->getText();
+        }
+
```

Comment:
Something like this?

$quoteTransfer->getEasycreditLegalText() ? $quoteTransfer->getEasycreditLegalText()->getText() : null;